packages/sanity/src/core/studio/workspaces/WorkspaceValidationError.ts

```diff
@@ -17,6 +17,6 @@ export class WorkspaceValidationError extends Error {
   constructor(message: string, options?: WorkspaceValidationErrorOptions) {
     super(message)
     this.index = options?.index
-    this.identifier = options?.workspace && getWorkspaceIdentifier(options.workspace, options.index)
+    this.identifier = options?.workspace && getIdentifier(options.workspace, options.index)
```

Comment:
Talking about generic names, `getWorkspaceIdentifier` -> `getIdentifier` seems to widen it a bit much no? I'd argue that getWorkspaceIdentifier was probably "safer" as an export.

Reply:
Not sure what happened here, will revert 👍